src/handlers.js

```diff
@@ -1,4 +1,47 @@
 // 1. handler to return the homepage('index.html and all resources from
-      //index.html')
+//index.html')
 // 2. handles search --> making a call to request.js and processes the result;
 // 3. handles other and returns 404 error.
+const path = require("path");
+const fs = require('fs');
+const apiRequest = require('./apiRequest');
+const router = require('./router.js');
+
+let handlers = module.exports = {}
+
+handlers.serveLanding = (request, response) => {
+  fs.readFile(path.join(__dirname, '..', 'public', 'index.html'), (error, file) => {
+    if (error) throw error;
```

Comment:
to my understanding, throwing an error in a callback like this would crash everything if triggered. we should be doing something useful with the error, perhaps writing a different status code and sending the response back with an error message to the front end?